concrete/src/User/UserList.php

```diff
@@ -156,6 +156,20 @@ public function filterByIsActive($isActive)
         $this->query->setParameter('uIsActive', $isActive);
     }
 
+    /**
+    * Filter list by whether a user is validated or not
+    * @param $val
+    */
+    public function filterByIsValidated($isValidated)
+    {
+        $this->includeInactiveUsers();
+        if (!$isValidated) {
+          $this->includeUnvalidatedUsers();
+          $this->query->andWhere('u.uIsValidated = :uIsValidated');
+          $this->query->setParameter('uIsValidated', $isValidated);
```

Comment:
On these three lines above the indent inside the `if` is only 2 spaces (should be 4 spaces as per PSR-2).